core/src/main/java/com/afollestad/materialdialogs/MaterialDialog.java

```diff
@@ -766,6 +766,8 @@ public Builder items(@NonNull Collection collection) {
                     i++;
                 }
                 items(array);
+            }else if(collection.size()==0){
```

Comment:
Standard formatting needs to be used here. 

```java
} else i f(collection.size() == 0 ) {
```

Android Studio's Reformat Code function will do that for you.